app/views/environments/welcome.html.erb

```diff
@@ -11,4 +11,8 @@
   <div class="blank-slate-pf-main-action">
       <%= new_link(_('New Puppet environment'), {}, { :class => "btn-lg" }) %>
   </div>
+  <p><%= _('Or') %></p>
```

Comment:
If there's no proxy I see 'Or' and nothing below that. It should check for at least one proxy to display this part. Otherwise looks good.

![envs](https://cloud.githubusercontent.com/assets/109773/15107657/4461f2b6-15d1-11e6-9761-871f5a84361f.png)